pyproject.toml

```diff
@@ -30,7 +30,7 @@ classifiers = [
 ]
 requires-python = ">=3.9"
 dependencies = [
-    "importlib_resources; python_version < '3.10'",
```

Comment:
Should remove this line if min-version is set to 3.10, the test action fail before is because the Python 3.9 test matrix is not removed: https://github.com/tatuylonen/wiktextract/blob/1e6ea054d3ae18a1c4fc688356675072c4f259f6/.github/workflows/test.yml#L30

Reply:
Yeah, sorry, I tried out stuff earlier and completely forgot these...

Reply:
I reverted this branch to 3.9, switching to 3.10 should be a separate operation.